### PR TITLE
fix(config): distinguis Popp Flow Stop versions 1 and 2

### LINKED ISSUE #2108: Request missing config for POPP Flow Stop 2 (POPE701479)
**Which device is missing?**
POPP Flow Stop 2 - POPE701479

**What are the IDs?**

Manufacturer ID: 0x0154  
Product type: 0x0005
Product ID: 0x0003

**Did you check if we really don't have it?**

- [x] It is not in the [configuration DB](https://devices.zwave-js.io/)
- [x] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)
- [x] It is not in the [TODO list](https://github.com/zwave-js/node-zwave-js/issues/1600)

**Do you have a link to the PDF manual?**
- [Manual](http://manual.zwave.eu/backend/make.php?lang=EN&sku=POPE701479&type=popp)
- [Product page](https://www.popp.eu/products/flow-stop-2/#1463610695891-082b0cba-747b)
- [Flyer](https://www.popp.eu//wp-content/uploads/2021/01/EN_Flow-Stop-2.pdf)
- [Data sheet](https://www.popp.eu//wp-content/uploads/2021/01/EN_Datenblatt_Flow-Stop-2.pdf)

### PR DESCRIPTION
**Pull request by marcus-j-davies**
Fixes: https://github.com/zwave-js/node-zwave-js/issues/2108.

Changes:
 - Fix Product/Type ID
 - Remove supportsZwavePlus

### CODE REVIEW

**Comment by marcus-j-davies** [comment]
@zwave-js-bot fix lint

**Comment by jmgiaever** [comment]
Flow Stop **2** do have Zwave+ though?

**Comment by marcus-j-davies** [comment]
Its picked up during interview. 👍

**Comment by jmgiaever** [comment]
> Its picked up during interview.

Ah, ok. Didn't know. Thanks mate! Kudos!

File: packages/config/config/devices/0x0154/flow_stop.json [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-// Popp 09501 Flow Stop
+// Popp & Co Flow Stop
+// Version 1 & 2
 {
 	"manufacturer": "Popp & Co",
 	"manufacturerId": "0x0154",
-	"label": "009501",
-	"description": "Popp 09501 Flow Stop",
+	"label": "Flow Stop",
+	"description": "Popp Flow Stop Version 1 & 2",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0154/flow_stop.json]: The description isn't meant to just repeat the manufacturer and label. If there's nothing to add, remove it.

File: packages/config/config/devices/0x0154/flow_stop.json [hunk 2]
```diff
@@ -1,25 +1,24 @@
-// Popp & Co 009501
-// Popp 09501 Flow Stop
+// Popp & Co Flow Stop
+// Version 1 & 2
 {
 	"manufacturer": "Popp & Co",
 	"manufacturerId": "0x0154",
-	"label": "009501",
-	"description": "Popp 09501 Flow Stop",
+	"label": "Flow Stop",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0154/flow_stop.json]: AFAIK Popp markets these as two different devices with two different labels (Flow Stop / Flow Stop 2). I think we should have two separate files with different labels for them - if the parameters are identical, you can `$import` them into the Flow Stop 2 file.

Comment (AlCalzone) [reply packages/config/config/devices/0x0154/flow_stop.json]: Ideally template the whole folder :)

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by marcus-j-davies** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0154/flow_stop.json (hunk 2 above)
Comment (marcus-j-davies) [reply packages/config/config/devices/0x0154/flow_stop.json]: As the saying goes: Learn to walk before you can run 😅 
Lint failed when `description` was removed, I set it to an empty string.

File: packages/config/config/devices/0x0154/flow_stop.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Popp & Co Flow Stop
+// Version 1
+{
+	"manufacturer": "Popp & Co",
+	"manufacturerId": "0x0154",
+	"label": "Flow Stop",
+	"description": "",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0154/flow_stop.json]:
```suggestion
```

File: packages/config/config/devices/0x0154/flow_stop_2.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// Popp & Co Flow Stop
+// Version 2
+{
+	"manufacturer": "Popp & Co",
+	"manufacturerId": "0x0154",
+	"label": "Flow Stop 2",
+	"description": "",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0154/flow_stop_2.json]:
```suggestion
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks good otherwise. I'll check with @blhoward2 how we want to go about the partial templating.

File: packages/config/config/devices/0x0154/flow_stop.json (hunk 3 above)
Comment (marcus-j-davies) [reply packages/config/config/devices/0x0154/flow_stop.json]: Lint failed doing this?

File: packages/config/config/devices/0x0154/flow_stop_2.json (hunk 4 above)
Comment (marcus-j-davies) [reply packages/config/config/devices/0x0154/flow_stop_2.json]: I'm guessing you can ignore lint failures? ,as it failed when I removed description.

File: packages/config/config/devices/0x0154/flow_stop.json (hunk 3 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0154/flow_stop.json]: Let me check

Comment (AlCalzone) [reply packages/config/config/devices/0x0154/flow_stop.json]: You're right. The description is enforced, but the style guide mentions it should be omitted - that's inconsistent. Empty description is bad though 😅

Comment (AlCalzone) [reply packages/config/config/devices/0x0154/flow_stop.json:7]: Let's make the description `"Valve Actuator"` then.

File: packages/config/config/devices/0x0154/flow_stop_2.json (hunk 4 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0154/flow_stop_2.json]: The model # is POPE009501. That should be the label. The description is Flow Stop.

File: packages/config/config/devices/0x0154/flow_stop.json (hunk 3 above)
Comment (marcus-j-davies) [reply packages/config/config/devices/0x0154/flow_stop.json]: Should I just prefix the description for now, or you going to tackle the linter?

Comment (AlCalzone) [inline packages/config/config/devices/0x0154/flow_stop.json]:
```suggestion
	"label": "POPE009501",
	"description": "Flow Stop",
```

File: packages/config/config/devices/0x0154/flow_stop_2.json (hunk 4 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0154/flow_stop_2.json]: That's the other one ;)

Comment (AlCalzone) [inline packages/config/config/devices/0x0154/flow_stop_2.json]:
```suggestion
	"label": "POPE701479",
	"description": "Flow Stop 2",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0154/flow_stop_2.json (hunk 4 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0154/flow_stop_2.json]: On my phone and missed that. Version 2 is POPP-701479-862

Comment (blhoward2) [reply packages/config/config/devices/0x0154/flow_stop_2.json]: It's popp not pope for the second one.

Comment (AlCalzone) [reply packages/config/config/devices/0x0154/flow_stop_2.json]: Sure?
http://manuals-backend.z-wave.info/make.php?lang=DE&sku=POPE701479&cert=ZC08-14030002

**Comment by jmgiaever** [comment]
It says POPE on the pamphlet in there box i received it in. 👍On Mar 19, 2021 3:33 PM, AlCalzone ***@***.***> wrote:
@AlCalzone commented on this pull request.



In packages/config/config/devices/0x0154/flow_stop_2.json:
> +	"label": "Flow Stop 2",
+	"description": "",

Sure?
http://manuals-backend.z-wave.info/make.php?lang=DE&sku=POPE701479&cert=ZC08-14030002

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**